nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -109,7 +121,8 @@ def add_sdk_refs_from_sdk_tags(doc, dependency_set, project_file)
           end
         end
 
-        def build_dependency(name, req, version, prop_name, project_file)
+        # rubocop:disable Metrics/ParameterLists
+        def build_dependency(name, req, version, prop_name, project_file, dev = false)
```

Comment:
We would usually go for a keyword arg here:
```suggestion
        def build_dependency(name, req, version, prop_name, project_file, dev: false)
```